app/src/main/res/values/styles.xml

```diff
@@ -11,7 +11,8 @@
 
     <!-- The launcher theme. It sets the main window background to the launch_screen drawable -->
     <style name="AppTheme.Launcher">
-        <item name="android:windowBackground">@drawable/launch_screen</item>
+        <!-- TODO: ADD THIS LINE BACK IN -->
+        <!--<item name="android:windowBackground">@drawable/launch_screen</item> -->
```

Comment:
Add this line back in, so master does not have the theme removed. 

In the future it might be easer to just change the 
`android:theme="@style/AppTheme.Launcher" >`
line in the manifest file to 
`android:theme="@style/AppTheme" >` to temporarily remove the splash screen